### PR TITLE
InDegree and InDegreeAV

### PR DESCRIPTION
**Pull request by c-yuwei**

### CODE REVIEW

File: braph2genesis/measures/_InDegree.gen.m [hunk 1]
```diff
@@ -0,0 +1,165 @@
+%% ¡header!
+InDegree < Measure (m, degree) is the graph degree.
```
Comment (annacanal) [inline braph2genesis/measures/_InDegree.gen.m:2]: in-degree

File: braph2genesis/measures/_InDegree.gen.m [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    B11     B12  
+    B21     B22
+    };
+
+known_in_degree = { 
+                  [1 1 1]'
+                  [2 2 1]'
+                  };
+
+g = MultiplexGraphBD('B', {B});
```
Comment (annacanal) [inline braph2genesis/measures/_InDegree.gen.m:117]: g = MultiplexGraphBD('B', B);
B is already a cell here

File: braph2genesis/measures/_InDegree.gen.m [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    B11 B12  
+    B21 B22
+    };
+
+known_in_degree = {
+                  [1 1 2]'
+                  [1 2 2]'
+                  };
+
+g = MultiplexGraphWD('B', {B});
```
Comment (annacanal) [inline braph2genesis/measures/_InDegree.gen.m:155]: g = MultiplexGraphBD('B', B);
B is already a cell here

File: braph2genesis/measures/_InDegreeAv.gen.m [hunk 4]
```diff
@@ -0,0 +1,162 @@
+%% ¡header!
+InDegreeAv < InDegree (m, degree) is the graph degree.
```
Comment (annacanal) [inline braph2genesis/measures/_InDegreeAv.gen.m:2]: is the graph average in-degree.

File: braph2genesis/measures/_InDegreeAv.gen.m [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+%%% ¡compatible_graphs!
+GraphWD
+GraphBD
+MultiplexGraphWD
+MultiplexGraphBD
+
+%% ¡props_update!
+
+%%% ¡prop!
+M (result, cell) is the in-degree-av.
```
Comment (annacanal) [inline braph2genesis/measures/_InDegreeAv.gen.m:25]: is the average in-degree

File: braph2genesis/measures/_InDegreeAv.gen.m [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+    B11 B12  
+    B21 B22
+    };
+
+known_in_degree_av = { 
+                     mean([1 1 1]')
+                     mean([2 2 1]')
+                     };
+
+g = MultiplexGraphBD('B', {B});
```
Comment (annacanal) [inline braph2genesis/measures/_InDegreeAv.gen.m:114]: same than before, B not {B}

File: braph2genesis/measures/_InDegreeAv.gen.m [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+    B11     B12  
+    B21     B22
+    };
+
+known_in_degree_av = {
+                     mean([1 1 2]')
+                     mean([1 2 2]')
+                     };
+
+g = MultiplexGraphWD('B', {B});
```
Comment (annacanal) [inline braph2genesis/measures/_InDegreeAv.gen.m:152]: same than before, B not {B}

**Review by annacanal — CHANGES_REQUESTED** [review]

File: braph2genesis/measures/_InDegreeAv.gen.m (hunk 4 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegreeAv.gen.m]: d5b4c69f

File: braph2genesis/measures/_InDegree.gen.m (hunk 1 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegree.gen.m]: d5b4c69f

File: braph2genesis/measures/_InDegree.gen.m (hunk 2 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegree.gen.m]: facdda24

File: braph2genesis/measures/_InDegree.gen.m (hunk 3 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegree.gen.m]: facdda24

File: braph2genesis/measures/_InDegreeAv.gen.m (hunk 4 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegreeAv.gen.m]: facdda24

File: braph2genesis/measures/_InDegreeAv.gen.m (hunk 5 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegreeAv.gen.m]: be2c4e02

File: braph2genesis/measures/_InDegreeAv.gen.m (hunk 6 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegreeAv.gen.m]: facdda2

File: braph2genesis/measures/_InDegreeAv.gen.m (hunk 7 above)
Comment (c-yuwei) [reply braph2genesis/measures/_InDegreeAv.gen.m]: facdda2

**Review by egolol — APPROVED** [review]

**Review by annacanal — APPROVED** [review]

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**